src/System-Platforms/KeyboardKey.class.st

```diff
@@ -1097,6 +1097,12 @@ KeyboardKey class >> metaRight [
 	^ self named: #META_R
 ]
 
+{ #category : #'accessing - keys - special' }
+KeyboardKey class >> menu [
+
+	^ self named: 'MENU'
```

Comment:
Most likely, 'MENU' should be replaced with #MENU. At least in other similar methods, a symbol is used, not a string.
```suggestion
	^ self named: #MENU
```